### PR TITLE
Add pageheader component

### PR DESCRIPTION
**Pull request by jnmarcus**
screen shot 2016-02-16 at 8 45 51 pm
screen shot 2016-02-16 at 8 47 47 pm

### CODE REVIEW

File: app/components/common/pageHeader/pageHeader.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(function() {
+  'use strict';
+
+  //var pageHeaderModule = require('./pageHeader');
+  var pageHeaderDirective = require('./pageHeaderDirective');
+
+  module.exports = angular.module('pageHeaderModule', [])
+    .directive('pageHeader', pageHeaderDirective);
+
+})();
```
Comment (adborden) [inline app/components/common/pageHeader/pageHeader.js]: You don't need the IIFE, browserify takes care of it for you.

File: app/components/common/pageHeader/pageHeaderDirective.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        nextElectionDate: '@nextElection'
+      },
+      link: link,
+      template: template,
+      controller: function() {} //if access to the controller is needed, add the controller in separate file
+    };
+    return directive;
+
+    function link(scope, el, attrs, vm) {
+      el.addClass('cfa-pageHeader');
```
Comment (adborden) [inline app/components/common/pageHeader/pageHeaderDirective.js]: Why the need for this class?

**Comment by adborden** [comment]
Cool, do you want to try swapping out some of the headers on the existing pages?

**Comment by adborden** [comment]
Also there's a linting error:

```
app/components/common/pageHeader/pageHeaderDirective.js
  line 22  col 37  'vm' is defined but never used.
  line 22  col 30  'attrs' is defined but never used.

✖ 2 problems
```

File: app/components/common/pageHeader/pageHeaderDirective.js (hunk 2 above)
Comment (jnmarcus) [inline app/components/common/pageHeader/pageHeaderDirective.js]: bc I forgot to commit the change where I added 'cfa-' to the 'pageHeader' class :open_mouth: 

It's updated now.

**Comment by jnmarcus** [comment]
@adborden I think I'll swap out the headers in a different pull request, just to keep things simple.  You're also more than welcome to add them in as well, altho there's no docs yet for it bc I haven't had time to get ngDoc up and running yet :disappointed: 

I took care of the linting "errors", so I'm going to go ahead and merge this in..

**Pull request merged**